### PR TITLE
otpilot: Add support for Manticore Vendor firmware information.

### PR DESCRIPTION
**Pull request by osenft**

### CODE REVIEW

File: userspace/otpilot/src/main.rs [hunk 1]
```diff
@@ -69,6 +72,20 @@ fn run_host_helper_demo() -> TockResult<()> {
     Ok(())
 }
 
+fn store_build_info(id: &str, segment_info: SegmentInfo, mut buf: &mut[u8]) {
```
Comment (dann21) [inline userspace/otpilot/src/main.rs]: Maybe add a comment that `id` is only used for error-spewage?

**Review by dann21 — APPROVED** [review]
LGTM

File: userspace/otpilot/src/main.rs (hunk 1 above)
Comment (osenft) [reply userspace/otpilot/src/main.rs]: That parameter was actually redundant. I replaced it with code.

**Review by dann21 — APPROVED** [review]
LGTM

**Pull request merged**